raiden/utils/ethereum_clients.py

```diff
@@ -24,6 +25,32 @@ def parse_geth_version(client_version: str) -> Optional[tuple]:
     return parse_version(matches.groups()[0])
 
 
+def support_check(
+    our_version: Version,
+    highest_supported_version_string: str,
+    lowest_supported_version_string: str,
+) -> bool:
+    if len(our_version._key[1]) != 3:
+        # This can happen for versions such as 0.0.0
+        return False
+
+    # our_major_num = our_version._key[1][0]
```

Comment:
Are these comments still necessary?

Reply:
Definitely not .... it's left over commented code, thank you for spotting it!